kernelci.org/content/en/docs/org/tsc/votes.md

```diff
@@ -37,6 +38,32 @@ outcome and a more detailed section listing each individual vote.
 
 ## Votes
 
+### 2024-05-13
+
+**Motion: Cleanup admin permissions**
+
+Amend the following permissions for Guillaume Tucker, as he's no longer
+closely involved with KernelCI:
```

Comment:
That's a bit subjective, I'm still on the TSC and taking part in some discussions.  It would be a bit more accurate to say that I've handed over my admin roles to other people.

Reply:
Rephrased, please check!